config/docker/nginx/dev/nginx-default.conf

```diff
@@ -40,5 +40,8 @@ server {
 
     location ~ ^/ {
         proxy_pass http://127.0.0.1:5173;
+        proxy_http_version 1.1;
```

Comment:
Its for HMR?

Reply:
Yep, i call commit - feat: :sparkles: **add HMR support,** upd yarn version
a8d7e812745d0da8ad2d071c8f655567415f28b4